src/software/backend/wifi_backend.h

```diff
@@ -54,4 +56,11 @@ class WifiBackend : public Backend
     std::unique_ptr<ThreadedProtoUdpListener<TbotsProto::RobotStatus>> robot_status_input;
     std::unique_ptr<ThreadedProtoUdpListener<TbotsProto::RobotLog>> robot_log_input;
     std::unique_ptr<ThreadedProtoUdpSender<DefendingSideProto>> defending_side_output;
+
+
+    std::unique_ptr<ThreadedEstopReader> estop_reader;
+
+    // the port the arduino is connected to. Can be found by running the command
+    // "pio device list"
+    static constexpr auto ARDUINO_PORT = "/dev/ttyACM0";
```

Comment:
should this be dynamic parameter?

Reply:
is it ok for it to be a dynamic parameter if changing the parameter has no effect unless the backend is restarted?

Reply:
we can make the dynamic parameter a command line argument